match/lib/match/importer.rb

```diff
@@ -82,29 +82,38 @@ def import_cert(params, cert_path: nil, p12_path: nil, profile_path: nil)
       output_dir_certs = File.join(storage.prefixed_working_directory, "certs", cert_type.to_s)
       output_dir_profiles = File.join(storage.prefixed_working_directory, "profiles", prov_type.to_s)
 
-      # Need to get the cert id by comparing base64 encoded cert content with certificate content from the API responses
-      token = api_token(params)
-      if token
-        UI.message("Creating authorization token for App Store Connect API")
-        Spaceship::ConnectAPI.token = token
+      should_skip_certificate_matching = params[:skip_certificate_matching]
+      # In case there is no access to Apple Developer portal but we have the certificates, keys and profiles
+      if should_skip_certificate_matching
+        # Make dir if doesn't exist
+        FileUtils.mkdir_p(output_dir_certs)
+        dest_cert_path = File.join(output_dir_certs, "certificate.cer")
+        dest_p12_path = File.join(output_dir_certs, "certificate.p12")
```

Comment:
Yeah, great Idea. I was a bit nervous too but didn't find a reason to have multiple distribution certificates. Maybe I did't think it trough so well.

Thank you and I understand that you have a lot going and that's why I didn't push for this too much :)

Reply:
Hi @joshdholtz @DunbarXX 

Nice find just an idea but could we instead of needing to specify the name for the certificate + p12 files just generate a UUID or random string and use this instead? For me I feel this would easier to work with going forward.

And means we only need to pass in a boolean parameter to skip cert validation with Apple.

Let me know your thoughts?

Reply:
I'd be okay with that if that works for you and other users! The only reason I thought of passing in a name is so its helpful which team or something the cert is actually for 🙃